### ISSUE TITLE
Inactivity timeout causes exception

### ISSUE THREAD

**Issue by qnm** [dparnell/homebridge-samsung-cac#1]
Firstly, thanks for this package, along with `samsung-cac` - they are excellent.

I'm seeing a bug when the activity times out, as per b8da24f3ea5fd10e5af95672d140ee9d8db92a8e

```
[21/09/2019, 14:32:31] TypeError: d.disconnect is not a function
    at Timeout.timeout.setTimeout [as _onTimeout] (/usr/lib/node_modules/homebridge-samsung-cac/index.js:93:15)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
```

I'm guessing that `d` is a `Device` but `disconnect()` is method on `Connection`, but I'm not yet very familiar with the code.

Thanks again.

**Comment by dparnell** [comment]
Hi, I'll take a look and see what I can see.

**Comment by dparnell** [comment]
I'm seeing it in my logs also. I'll see what I can do about it

**Comment by dparnell** [comment]
This should be fixed in 0.0.4 now. Let me know and I will close this issue
